### DISCUSSION TITLE
SQLite Viewer Browser Extension portraits itself as DB Browser for SQLite in its description

### DISCUSSION THREAD

**Discussion by FaceCrap** [sqlitebrowser/sqlitebrowser#3824] [General]
As the subject says, the SQLite Viewer browser extension lists itself as being DB Browser for SQLite in its description on the [Microsoft Edge Addon site](https://microsoftedge.microsoft.com/addons/detail/sqlite-viewer/maodolcbcnpdjeiklhembjpidlnapghf)

Looks very much like the developer of this extension is trying to piggyback on the popularity of this great tool.
Not only that, but it also gives users the wrong impression that it might be associated with DB Browser for SQLite since that's how it turns up in search results when searching for extensions for the standalone version.

![image](https://github.com/user-attachments/assets/fa1c9fad-33d2-408c-b794-2d42b0cc2606)

**Comment by justinclift** [comment]
Thanks for letting us know.  I've just reported that extension via the "Report Abuse" link on that page:

![image](https://github.com/user-attachments/assets/08bb420c-7ef7-46e2-8246-1225f642f323)
